Client/Frontend/Browser/Playlist/PlaylistPadViewController.swift

```diff
@@ -0,0 +1,950 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import UIKit
+import AVKit
+import AVFoundation
+import CoreData
+import SnapKit
+import BraveShared
+import Shared
+
+private let log = Logger.browserLogger
+
+private protocol PlaylistPadControllerDetailDelegate: class {
+    func setCurrentItemIndex(_ index: Int)
+    func getCurrentItemIndex() -> Int
+    func updateNowPlayingMediaInfo(_ info: PlaylistInfo?)
+    func updateNowPlayingMediaArtwork(image: UIImage?)
+    func setControlsEnabled(_ enabled: Bool)
+    func updatePlayerControlsState()
+    func loadMediaItem(_ item: PlaylistInfo, index: Int, autoPlayEnabled: Bool, completion: @escaping (PlaylistMediaInfo.MediaPlaybackError) -> Void)
+    func displayLoadingResourceError()
+    func play()
+    func stop()
+}
+
+class PlaylistPadViewController: UIViewController {
```

Comment:
Should not use a custom controller for iPad only, as iPad can be viewed in other size classes